pkg/query-service/app/http_handler.go

```diff
@@ -3515,6 +3532,25 @@ func (aH *APIHandler) getAllOrgPreferences(
 	aH.Preference.GetAllOrgPreferences(w, r)
 }
 
+// Quick Filters - Registers/Upserts all Quick Filters based on Signals
+func (aH *APIHandler) getAllQuickFilters(
+	w http.ResponseWriter, r *http.Request,
+) {
+	aH.QuickFilters.GetOrgQuickFilters(w, r)
```

Comment:
no error handling for these ?

Reply:
Error handling on the http_handler level? I have done the error handling inside the function itself

Reply:
ahh right not required. Please resolve the pr conflicts.